config/config.exs

```diff
@@ -137,6 +137,10 @@ config :skate, Oban,
     }
   ]
 
+# Include 2 logger backends
+config :logger,
+  backends: [:console, Sentry.LoggerBackend]
```

Comment:
question: Hmm, we don't have Sentry in local dev or test, should this be in `prod.exs` instead?

Reply:
Yeah, I waffled on this, because prod has the dsn check and would prevent anything in local dev / test from getting sent to Sentry anyway. But I don't mind moving it either

Reply:
Oh that's a great point actually, thanks! I now realize that if it's in the base config, that means you can test sentry locally by providing the variable, and have it disabled by default, like it would be in a deployed environment. I do like that.

So now I don't really have a strong opinion on where it should go 🫤 I'd be happy with dropping that commit if you wanted.